staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -465,8 +476,8 @@ type namedCreaterAdapter struct {
 	rest.Creater
 }
 
-func (c *namedCreaterAdapter) Create(ctx request.Context, name string, obj runtime.Object) (runtime.Object, error) {
-	return c.Creater.Create(ctx, obj)
+func (c *namedCreaterAdapter) Create(ctx request.Context, name string, obj runtime.Object, includeUninitialized bool) (runtime.Object, error) {
```

Comment:
I honestly don't have any idea what to expect a parameter called "includeUnititialized" to Create to do.